GitHub-Security-Settings.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Organization and Repository Settings
+
+This documents which settings are used for Mozilla IAM repositories, and why.
+All repositories created should follow the rules established here unless otherwise stated.
+
+See also (CONTRIBUTING.md)[CONTRIBUTING.md] for more information about contributing code to Mozilla IAM repositories. 
+
+## mozilla-iam GitHub organization owners
+
+The GitHub "org owners" for https://github.com/mozilla-iam/ are required to:
+
+- Use a separate GitHub account from their main GitHub account, dedicated to org management. This account may not be used to commit code and must have 2FA enabled.
+- Follow rules documented in this document.
+
+### Teams and access control
+URL: https://github.com/orgs/mozilla-iam/teams
+
+An "org owner" may add users to teams. The teams should have a description which indicates their intended purpose.
+
+- No user may be added to teams that will be given universal `write` access to all repository by default.
```

Comment:
I suspect these can change over time (like the unito bot we have now)

Reply:
The unito bot though (or specifically the team containing the unito bot) would *not* be a team given universal `write` access though right? Meaning this would *not* be a team that would be listed here.